pkg/hubble/container/ring.go

```diff
@@ -31,6 +32,67 @@ var (
 	ErrInvalidRead = errors.New("read position is no longer valid")
 )
 
+// capacity implements Capacity.
```

Comment:
nit: shouldn't the public interface `Capacity` be declared/defined before this comment? Just reading this in order is weird as this comment refer to something that is defined after. Not critical though.